pyproject.toml

```
@@ -1,3 +1,56 @@
+[project]
+name = "SPRAS"
+version = "v0.0.1"
+description = "Signaling Pathway Reconstruction Analysis Streamliner (SPRAS)"
+authors = [
+  { name = "Anthony Gitter", email = "agitter@morgridge.org" },
+]
+license = { file = "LICENSE" }
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+requires-python = ">=3.8"
+dependencies = [
+    "adjusttext==0.7.3",
+    "snakemake==7.19.1",
+    "docker==5.0.3", # Switched from docker-py to docker because docker-py is not maintained in pypi. This appears to have no effect
+    "matplotlib==3.5",
+    "networkx==2.8",
+    "pandas==1.4",
+    "pip==22.1",
+    "requests==2.28",
+    "scikit-learn==1.2", #Version bumped from 1.1-->1.2 to get past compilation errors. No other errors encountered
+    "seaborn==0.12",
+    "spython==0.2",
+    # Only required for GraphSpace
+    "commonmark==0.9",
+    "docutils", #==0.18
+    "jinja2==3.1",
+    "mock==4.0",
+    "recommonmark==0.7",
+    "sphinx==5.0",
+    "graphspace_python==1.3.1",
+    "sphinx-rtd-theme==1.0.0",
+]
+
+[project.optional-dependencies]
+dev = [
+   # Only required for development
+    "pre-commit==2.20",
+    "pytest==7.1",
+]
+
+[project.urls]
+"Homepage" = "https://github.com/Reed-CompBio/spras"
+"Bug Tracker" = "https://github.com/Reed-CompBio/spras/issues"
```

```suggestion
"Issues" = "https://github.com/Reed-CompBio/spras/issues"
```

I prefer a more generic term because we use issues for bugs and general discussion about the project